en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md

```diff
@@ -14,16 +14,57 @@ Let's learn how Sam implements this!
 
 Follow the steps below to configure account locking due to failed login attempts.
 
-1.	Open the `deployment.toml` file in the `<IS_HOME>/repository/conf` directory and check whether the following listener configs are in place.
+1.	Open the `deployment.toml` file in the `<IS_HOME>/repository/conf` directory.
 
-	```
-	[event.default_listener.identity_mgt]
-	priority= "50"
-	enable = false
-	[event.default_listener.governance_identity_mgt]
-	priority= "95"
-	enable = true
-	```
+	1.	To request password entry from the users who are registered via Management Console, add the following configurations. 
+
+
+	 	```
+	  	[event.default_listener.identity_mgt]
+	  	priority= "50"
+	  	enable = false
+	  	event.default_listener.governance_identity_mgt]
+	    priority= "95"
```

Comment:
This looks misaligned. Does it render properly?

Reply:
<img width="723" alt="Screenshot 2022-05-22 at 21 48 29" src="https://user-images.githubusercontent.com/40081059/169705352-595aaac4-1708-4161-b0d6-6da643dc000e.png">

**Renders properly and properly aligned in the text-editor as well**